src/app/shared/components/config-cards/twitchbot-config/twitchbot-config.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="title-row">
+  <h2 class="mat-h2 title-name">Twitch Bot Config</h2>
+
+  <mat-icon svgIcon="twitch"></mat-icon>
+</div>
+
+<div>
+  <form [formGroup]="form">
+    <p>To obtain the bot token, login into twitch with your bot account, open <a href="https://twitchapps.com/tmi/">this URL</a> and click on the connect button.<br/></p>
```

Comment:
could you change the link to a mat-button? in the current way its quite hard to see the name or that its even a link to click

Reply:
![memebox-twitch-bot-config](https://user-images.githubusercontent.com/2153382/99914693-5dfbd580-2cff-11eb-8040-58c4971e3ac2.png)

Is this ok?

Reply:
looks good!! gonna take a look in a local version too